lib/Onyx.js

```diff
@@ -1043,10 +1062,12 @@ function merge(key, changes) {
                 // After that we merge the batched changes with the existing value
                 let modifiedData = applyMerge(existingValue, [batchedChanges]);
```

Comment:
```suggestion
                const modifiedData = cleanObject(applyMerge(existingValue, [batchedChanges]));
```

Then you can remove the line below.

Reply:
I actually prefer the two-line approach more, since it's easier for readability imo. But i can also change this to your version 👍